build/package/homebrew/homebrew.template.rb

```diff
@@ -30,4 +29,7 @@ def install
     bin.install_symlink Dir["#{libexec}/bin/*"]
   end
 
+  test do
+    system "#{bin}/kbc --version"
```

Comment:
Pridal som tu este `test`, ktory otestuje funkcnost instalacie cez `brew`, ked dobehne.

Reply:
Nejdem teraz robit release, spravi sa release so vsetkymi novymi featurami, ked to pomergujeme/dokoncime.
Ak by tam bol este nejaky problem, tak sa to proste fixne potom.